index.js

```diff
@@ -1,156 +1,169 @@
             validate: (value) => {
-              if (value.startsWith("https://unsplash.com/collections/")) {
-                return true;
+              if (
+                value.startsWith(
+                  'https://unsplash.com/collections/'
+                )
+              ) {
+                return true
               }
 
-              return "🚨 Please enter a valid Unsplash Collections URL!";
+              return '🚨 Please enter a valid Unsplash Collections URL!'
             },
           },
         ])
-        .then((answers) => (options.collection = answers.collection));
+        .then((answers) => (options.collection = answers.collection))
     }
 
     await inquirer.prompt(firstQuestions).then((answers) => {
       for (let a in answers) {
-        options[a] = answers[a];
+        options[a] = answers[a]
       }
-    });
+    })
 
     if (options.random) {
-      if (options.orientation === "custom") {
+      if (options.orientation === 'custom') {
         await inquirer
           .prompt([
-            nextQuestions({ required: true, side: "width" }),
-            nextQuestions({ required: true, side: "height" }),
+            nextQuestions({ required: true, side: 'width', }),
+            nextQuestions({ required: true, side: 'height', }),
           ])
           .then((answers) => {
-            options.width = answers.width;
-            options.height = answers.height;
-          });
+            options.width = answers.width
+            options.height = answers.height
+          })
       } else {
         await inquirer
-          .prompt([nextQuestions({ required: false, side: "width" })])
+          .prompt([nextQuestions({ required: false, side: 'width', }),])
           .then((answers) => {
-            options.width = answers.width;
-          });
+            options.width = answers.width
+          })
         if (!options.width) {
           await inquirer
-            .prompt([nextQuestions({ required: false, side: "height" })])
+            .prompt([
+              nextQuestions({ required: false, side: 'height', }),
+            ])
             .then((answers) => {
-              options.height = answers.height;
-            });
+              options.height = answers.height
+            })
         }
       }
     }
 
-    if (options.orientation === "custom" || options.orientation === "mixed") {
-      delete options.orientation;
+    if (
+      options.orientation === 'custom' ||
+            options.orientation === 'mixed'
+    ) {
+      delete options.orientation
     }
 
     await inquirer
       .prompt([
         {
-          type: "confirm",
-          name: "saveCredits",
-          message: "🗂  Export the credits for the photos to a .json file?",
+          type: 'confirm',
+          name: 'saveCredits',
+          message:
+                        '🗂  Export the credits for the photos to a .json file?',
           default: true,
```

Comment:
the indentation here looks a bit odd; is this intentional?

Reply:
was some sort of eslint / prettier conflict on my end, i think - installing the plugin to make them behave better